### ISSUE TITLE
the link about "statechart extension" are missing

### ISSUE THREAD

**Issue by bahuasheng** [javapathfinder/jpf-core#173]
In https://github.com/javapathfinder/jpf-core/wiki/Different-applications-of-JPF, the link about "statechart extension" are missing

**Comment by cyrille-artho** [comment]
Yes, the link points to a non-existent page. Does anyone know the correct link?

**Comment by manish3499** [comment]
@cyrille-artho  http://babelfish.arc.nasa.gov/trac/jpf/wiki/projects/jpf-statechart
This could be the one, but it asks for username and password for authentication

**Comment by cyrille-artho** [comment]
Perhaps there is another site where the project now is? The content on babelfish is very outdated, and, as you noticed, no longer accessible to outside users.

**Comment by manish3499** [comment]
As far as I've seen, all the content related to JPF is found on GitHub, the bablefish page and the sourceforge.net page. The sourceforge.net page has a link ( http://javapathfinder.sourceforge.net/extensions/statechart/doc/index.html ) related to statechart extension, but that too returns a 404 error.

**Comment by cyrille-artho** [comment]
It is possible that this extension has not been maintained for a while, and that there is no working web page available anymore. That would be quite unfortunate.
The use of sourceforge predates even the Mercurial (hg) repository on babelfish, so that page is likely even older than the now-defunct one on babelfish.
